Product/SoapUI_Test/RegressionSuite/Standard/DocQuerytestsforUDDI/DocQuerytestsforUDDI-soapui-project.xml

```diff
@@ -5447,7 +5447,7 @@ count(//ns6:AdhocQueryResponse[1]/ns3:RegistryObjectList[1]/ns3:ExtrinsicObject)
         </con:property>
       </con:properties>
       <con:reportParameters/>
-      <con:tearDownScript>nhinc.FileUtils.restoreToMasterConfiguration(context, log);</con:tearDownScript>
+      <con:tearDownScript>nhinc.FileUtils.restoreConfiguration(context.findProperty("GatewayPropDir"), log);</con:tearDownScript>
```

Comment:
Gotcha. If thats the case, I would suggest keeping backupConfiguration() in setup script.